specifications/api/swagger.yaml

```diff
@@ -2748,6 +2748,86 @@ components:
 
         "none": *ApiNullStakeKey
 
+    ApiMintBurnTransaction: &ApiMintBurnTransaction
+      type: object
+      required:
+        - transaction
+        - monetary_policy_index
+        - policy_id
+        - asset_name
+        - subject
+      properties:
+        transaction: *ApiTransaction
+        monetary_policy_index: *derivationSegment
+        policy_id: *assetPolicyId
+        asset_name: *assetName
+        subject: *assetFingerprint
+
+    ApiAssetQuantity: &ApiAssetQuantity
+      type: object
+      required:
+        - quantity
+        - unit
+      properties:
+        quantity:
+          type: integer
+          example: 14
+        unit:
+          type: string
+          enum:
+            - assets
+
+    ApiMintData: &ApiMintData
+      type: object
+      required:
+        - receiving_address
+        - amount
+      properties:
+        receiving_address: *addressId
+        amount: *ApiAssetQuantity
+
+    ApiBurnData: &ApiBurnData
+      allOf:
+        - *ApiAssetQuantity
+
+    ApiMintBurnOperation: &ApiMintBurnOperation
```

Comment:
If I'm not mistaken operation is not quite correctly formatted in API doc:
![Screenshot from 2021-06-17 21-24-19](https://user-images.githubusercontent.com/42900201/122460597-98d29d00-cfb2-11eb-913f-d7d9d4242f9b.png)

It is:
```
   "operations": [
      {
        "mint": {
          "receiving_address": "addr1sjck9mdmfyhzvjhydcjllgj9vjvl522w0573ncustrrr2rg7h9azg4cyqd36yyd48t5ut72hgld0fg2xfvz82xgwh7wal6g2xt8n996s3xvu5g",
          "amount": {
            "quantity": 14,
            "unit": "assets"
          }
        },
        "burn": {
          "quantity": 14,
          "unit": "assets"
        }
      }
    ]
```
but shouldn't it be:
```
 [
      {
        "mint": {
          "receiving_address": "addr_test1qq223dyrt96yx79rz3yda2vs6xjx28ts2ax39hp8cxq2p0shpgwhzykzha0z6w27gxsps40gurvxjt2vxh26009q5wqqxpj2ey",
          "amount": {
            "quantity": 14,
            "unit": "assets"
          }
        } 
       },
       {
        "burn": {
          "quantity": 14,
          "unit": "assets"
        }
      }
]
```
i.e. "mint" and "burn" in separate `{}` ?

Reply:
Good eye @piotr-iohk, I've fixed this in a6b9895.

Thanks for adding the warning about metadata too!

Reply:
See new redoc output:

![mint](https://user-images.githubusercontent.com/5304556/122497932-d0782e00-d020-11eb-9ded-cc4fe1d136d7.png)
![burn](https://user-images.githubusercontent.com/5304556/122497939-d5d57880-d020-11eb-9068-c1851f0d26de.png)

Reply:
Nice, thanks!

Question: Are multiple `mint` or `burn` going to be allowed in single transaction?

`"operations": []` is an Array object, which implicates that it will be the case, just want to make sure.
On the other hand there is `Array one of` in the redoc, so it might be slightly confusing, what do you think? Maybe if there is only one mint or burn allowed they should be inside `"operations": {}` not `"operations": []`?

Reply:
We planned for multiple mint/burn operations in a single transaction. It's something supported by cardano-node and it wasn't hard to allow.